### ISSUE TITLE
[RFE] Allow running an specifig job list

### ISSUE THREAD

**Issue by rsevilla87** [cloud-bulldozer/kube-burner#371]
**Is your feature request related to a problem? Please describe.**

When kube-burner runs a benchmark composed of several jobs there's no way to tell kube-burner to run an specific job from the list. 

**Describe the solution you'd like**

Having a flag --job-list=J1,J2,J3 can be useful to make kube-burner to just execute those specific jobs in the given order

**Comment by vishnuchalla** [comment]
@rsevilla87 I am bit confused on the significance of adding this new flag. Instead of having this flag, can't we achieve the same by only specifying the jobs that are required to run in the `config.yaml` file and parse them in such a way to preserve their order and execute them accordingly?

**Comment by rsevilla87** [comment]
> @rsevilla87 I am bit confused on the significance of adding this new flag. Instead of having this flag, can't we achieve the same by only specifying the jobs that are required to run in the `config.yaml` file and parse them in such a way to preserve their order and execute them accordingly?

You're right, closing this issue, it's not very important and not very useful actually.
